src/data/zsqrtd/basic.lean

```diff
@@ -145,6 +145,9 @@ by simp [ext]
 @[simp] theorem muld_val (x y : ℤ) : sqrtd * ⟨x, y⟩ = ⟨d * y, x⟩ :=
 by simp [ext]
 
+@[simp] theorem dmuld : sqrtd * sqrtd = d :=
```

Comment:
```suggestion
@[simp] theorem sqrtd_mul_sqrtd : sqrtd * sqrtd = d :=
```

Reply:
Sure, although this file seems to use `muld` in a lot of other lemma names. Do you want to go with `sqrtd_mul_sqrtd` anyway?

Reply:
Oooh, I had not realized that. Well, let's leave it like this then